scripts/release_scripts/update_configs.py

```diff
@@ -167,6 +167,24 @@ def add_mailgun_api_key():
             f.write(line)
 
 
+def add_redishost():
+    """Adds redishost key to feconf.py."""
+    redishost = getpass.getpass(
+        prompt=('Enter REDISHOST from the release process doc.'))
```

Comment:
Trim spaces from each side, and validate that REDISHOST is in IP address format before continuing (%d.%d.%d.%d where each %d is 1-3 digits)

Reply:
Done.